### PR TITLE
esm: rename services

### PR DESCRIPTION
**Pull request by lucasmoura**
## Proposed Commit Message
esm: rename services

Change ESM services from UA Infra and UA Apps to Ubuntu Pro: ESM Infra and Ubuntu Pro: ESM Apps

## Test Steps
Verify the integration tests are working as expected

## Desired commit type

 - [ ] Squash and merge: Using the "Proposed Commit Message"
 - [ ] Create a merge commit and use "Proposed Commit Message"
 - [x] Rebase and merge, no merge commit, rely only on existing commit messages

## Checklist

 - [x] I have updated or added any unit tests accordingly
 - [x] I have updated or added any integration tests accordingly
 - [ ] I have updated or added any documentation accordingly

### CODE REVIEW

File: docs/make.bat [hunk 1]
```diff
@@ -0,0 +1,35 @@
+@ECHO OFF
```
Comment (CalvoM) [inline docs/make.bat:1]: Quick question, when is this file used?

Comment (lucasmoura) [reply docs/make.bat:1]: This is a mistake, I will remove that file

File: features/attached_commands.feature [hunk 2]
```diff
@@ -488,35 +488,35 @@ Feature: Command behaviour when attached to an UA subscription
...
-            You can find our more about the esm service at
-            https://ubuntu.com/security/esm
-            """
+        Help:
+        esm-infra provides access to a private ppa which includes available high
+        and critical CVE fixes for Ubuntu LTS packages in the Ubuntu Main
+        repository between the end of the standard Ubuntu LTS security
+        maintenance and its end of life. It is enabled by default with
+        Extended Security Maintenance (ESM) for Ubuntu Pro: ESM Apps and
+        Ubuntu Pro: ESM Infra. You can find our more about the esm service at
```
Comment (orndorffgrant) [inline features/attached_commands.feature:507]: The change makes this a bit confusing. How about something like this
```suggestion
        maintenance and its end of life. It is enabled by default with
        Ubuntu Pro. You can find our more about the esm service at
```

File: help_data.yaml [hunk 3]
```diff
@@ -15,10 +15,10 @@ cis:
 
 esm-apps:
     help: |
-      UA Apps: Extended Security Maintenance is enabled by default on entitled
-      workloads. It provides access to a private PPA which includes available
-      high and critical CVE fixes for Ubuntu LTS packages in the Ubuntu Main
-      and Ubuntu Universe repositories from the Ubuntu LTS release date until
+      Ubuntu Pro: Extended Security Maintenance (ESM) Apps is enabled by default
```
Comment (orndorffgrant) [inline help_data.yaml:18]: This is a combination of the long and short names. I think we should just use the long name
```suggestion
      Extended Security Maintenance for Applications is enabled by default
```

**Review by orndorffgrant — APPROVED** [review]
Two small suggestions for the help text but LGTM and I think good to merge when CI is happy

**Comment by lucasmoura** [comment]
@orndorffgrant updated

**Comment by orndorffgrant** [comment]
Can we also drop the "NO WARRANTY" message as part of this PR? @lucasmoura

**Comment by lucasmoura** [comment]
@orndorffgrant Done

File: features/attach_validtoken.feature [hunk 4]
```diff
@@ -143,28 +143,21 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
...
         Then if `<release>` in `xenial` and stdout matches regexp:
         """
-        \*Your UA Infra: ESM subscription has EXPIRED\*
-        Enabling UA Infra: ESM service would provide security updates for following packages:
+        \*Your Ubuntu Pro: ESM Infra subscription has EXPIRED\*
+        Enabling Ubuntu Pro: ESM Infra service would provide security updates for following packages:
           .*
         \d+ esm-infra security update\(s\) NOT APPLIED. Renew your UA services at
-        https:\/\/ubuntu.com\/advantage
+        https:\/\/ubuntu.com\/pro
```
Comment (orndorffgrant) [inline features/attach_validtoken.feature:160]: Do we want to make this change yet? @lucasmoura
We might release before the URL is live, in which case there will be a few days with a broken link - but maybe that's not that bad?

**Comment by renanrodrigo** [comment]
There are some more entries related to `UA Infra`, for instance - should those be tackled in this PR?
```
ubuntu@jammy:~/u-a-c/$ ack 'UA Infra'
docs/howtoguides/get_token_and_attach.md
20:This machine is now attached to 'UA Infra - Essential (Virtual)'

docs/explanations/motd_messages.md
21:  1 of these updates is a UA Infra: ESM security update.
30:  UA Infra: Extended Security Maintenance (ESM) is enabled.
34:  1 of these updates is a UA Infra: ESM security update.
46:  1 of these updates is a UA Infra: ESM security update.
51:  Learn more about enabling UA Infra: ESM service for Ubuntu 16.04 at

(...)

features/unattached_commands.feature
83:        UA Infra: Extended Security Maintenance \(ESM\) is not enabled.

features/attach_validtoken.feature
108:        \d+ of these updates (is|are) (a|an)? UA Infra: ESM security update(s)?.
132:        UA Infra: Extended Security Maintenance \(ESM\) is enabled.
135:        \d+ of these updates (is|are) UA Infra: ESM security update(s)?.
```

**Review by renanrodrigo — APPROVED** [review]
comments solved in direct conversation
lgtm

**Comment by lucasmoura** [comment]
@orndorffgrant Done

**Pull request merged**